sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_models.py

```diff
@@ -1321,11 +1327,18 @@ class RecognizePiiEntitiesAction(DictMixin):
 
     :keyword str model_version: The model version to use for the analysis.
     :keyword str domain_filter: An optional string to set the PII domain to include only a
-    subset of the PII entity categories. Possible values include 'phi' or None.
+        subset of the PII entity categories. Possible values include 'phi' or None.
     :keyword str string_index_type: Specifies the method used to interpret string offsets.
         `UnicodeCodePoint`, the Python encoding, is the default. To override the Python default,
         you can also pass in `Utf16CodePoint` or TextElements_v8`. For additional information
         see https://aka.ms/text-analytics-offsets
+    :ivar str model_version: The model version to use for the analysis.
+    :ivar str domain_filter: An optional string to set the PII domain to include only a
+        subset of the PII entity categories. Possible values include 'phi' or None.
+    :ivar str string_index_type: Specifies the method used to interpret string offsets.
```

Comment:
Just for future clarification, we are supposed to be duplicating the docs for keyword args and internal variables?

Reply:
yeah, if the model is used as an input we include the keyword arg docs in addition to the internal variables

Reply:
Ok, thanks for the info, I'll be sure to add both next time.  :)